linux_os/guide/services/docker/docker_storage_configured/rule.yml

```diff
@@ -18,8 +18,6 @@ rationale: |-
 
 severity: low
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Limitation of OpenSCAP. Content needs to work across all SCAP scanners accurately. Reverting this change.

Reply:
ditto (see `service_cockpit_disabled`)